main.js

```diff
@@ -60,6 +60,7 @@ let self = require('sdk/self');
 const RESOURCE_HOST = 'tabcenter';
 
 let observerService = Cc['@mozilla.org/observer-service;1'].getService(Ci.nsIObserverService);
+let startupFinishedObserver = {};
```

Comment:
I think we should start with `startupFinishedObserver` set to `null`, so that we don't try to remove it in the `unload` function.